### PR TITLE
GAT-9: Added test feature for Text Resize block.

### PR DESCRIPTION
**Pull request by barbun**
- Added default preparation to Drupal API scenarios to set window size.
- Added simple context functions to determine if element with CSS property exists on a page
- Added test scenario that covers the functionality of govcms_text_resize module

### CODE REVIEW

**Comment by srowlands** [comment]
test this

**Comment by barbun** [comment]
test this

**Comment by srowlands** [comment]
test this

**Comment by srowlands** [comment]
test this

**Comment by srowlands** [comment]
test this

**Comment by barbun** [comment]
test this

**Comment by srowlands** [comment]
test this

**Comment by srowlands** [comment]
test this

File: build/tests/behat/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -25,6 +25,15 @@ public function __construct() {
   }
 
   /**
+   * Set default browser window size.
+   *
+   * @BeforeScenario @drupal
+   */
+  public function resizeWindow() {
+    $this->getSession()->getDriver()->maximizeWindow();
```
Comment (NigelCunningham) [inline build/tests/behat/bootstrap/FeatureContext.php]: Could this be called maximizeWindow instead? resizeWindow implies the caller should have some control over the size (and presumably supply dimensions).

Comment (barbun) [inline build/tests/behat/bootstrap/FeatureContext.php]: Good point, agreed. Will change!

File: build/tests/behat/bootstrap/FeatureContext.php [hunk 2]
```diff
@@ -266,4 +275,27 @@ protected function selectUserVBOCheckbox($username) {
...
+   * Asserts that an element, specified by CSS selector, does not exist.
+   *
+   * @param string $selector
+   *   The CSS selector to search for.
+   *
+   * @Then the element :selector should not exist
+   */
+  public function theElementShouldNotExist($selector) {
+    $this->assertSession()->elementNotExists('css', $selector);
+  }
```
Comment (NigelCunningham) [inline build/tests/behat/bootstrap/FeatureContext.php]: Blank line before closing the class definition should be preserved.

Comment (barbun) [inline build/tests/behat/bootstrap/FeatureContext.php]: Ah, good catch!

Comment (barbun) [inline build/tests/behat/bootstrap/FeatureContext.php]: Resolved

**Pull request merged**